modules/lss-creditserweiterung/creditserweiterung.user.js

```diff
@@ -0,0 +1,157 @@
+            1000000: 'Lieutenant',
+            5000000: 'Captain',
+            20000000: 'Staff Captain',
+            50000000: 'Battalion Chief',
+            1000000000: 'Division Chief',
+            2000000000: 'Deputy Chief',
+            5000000000: 'Fire Chief'
+        },
+        texts:{
+            reloadMessage: 'Please open your profile<br>and reload the website!',
+            dropdownName: 'Creditsextension',
+            creditsOverview: 'Overview',
+            coinsProtokoll: 'Coins List',
+            earnedCredits: 'Totally earned Credits',
+            creditsToNextRank: 'Credits to next rank',
+            updateMessage: 'Values are updated<br>all 5 minutes.',
+            allianceFunds: 'Alliance Funds',
+            noFurtherRank: 'There are no further ranks'
+        }
+    };
+    I18n.translations.nl.lssm.creditserweiterung = {
+        earnedCreditsRegEx: /(.*)([0-9]\.)? verdiende Credits.*/i,
+        ranks: {
+            0: 'Aspirant',
+            200: 'Brandwacht (Manschap A)',
+            10000: 'Hoofdbrandwacht (Manschap B)',
+            100000: 'Brandmeester',
+            1000000: 'Hoofdbrandmeester',
+            5000000: 'Commandeur',
+            50000000: 'Adjunct-Hoofdcommandeur',
+            500000000: 'Hoofdcommandeur',
+            5000000000: 'Erelid'
+        },
+        texts:{
+            reloadMessage: 'Open uw profiel<br>en herlaad de website!',
+            dropdownName: 'Credits-uitbreiding',
+            creditsOverview: 'Overzicht statistiek',
+            coinsProtokoll: 'Coin protocol',
+            earnedCredits: 'Totaal aantal credits',
+            creditsToNextRank: 'Credits voor de volgende rang',
+            updateMessage: 'Waarden actualiseren<br>om de 5 minuten.',
+            allianceFunds: 'Teamkas',
+            noFurtherRank: 'Geen verdere rangen meer'
+        }
+    };
+
+    function getCreditsOfNextRank(earnedCredits){
+        let ranks = I18n.t('lssm.creditserweiterung.ranks');
+        for (var key in ranks) {
+            if(earnedCredits > key){
+                continue;
+            } else {
+                return key;
+            }
+        }
+        return null;
+    }
+
+    function getEarnedCredits() {
+        let response = $.ajax({
+            type: "GET",
+            url: $('#navbar_profile_link').attr('href'),
+            async: false
+        }).responseText;
+        let results = $(response).find('.page-header').text().trim().match(I18n.t('lssm.creditserweiterung.earnedCreditsRegEx'));
+        return parseInt(results[1].replace(/\./g,'').replace(/,/g, '')); // Delete all "." and "," as these are kilo-splitting-chars of DE, NL and EN
+    }
+
+    function getAlianceFundsCredits() {
+        let response = $.ajax({
+            type: "GET",
+            url: "./verband/kasse",
+            async: false
+        }).responseText;
+        return parseInt($(response).find('h1')[0].innerText.replace(/\./g, "").replace(/,/g, ""));
```

Comment:
Wenn die Verbandskasse deaktviert ist, wird leider für diese Zeile noch ein error ausgeworfen.
`creditserweiterung.user.js?uid=leitstellenspiel675:113 Uncaught TypeError: Cannot read property 'innerText' of undefined
    at getAlianceFundsCredits (creditserweiterung.user.js?uid=leitstellenspiel675:113)
    at updateCredits (creditserweiterung.user.js?uid=leitstellenspiel675:126)
    at creditserweiterung.user.js?uid=leitstellenspiel675:155
    at creditserweiterung.user.js?uid=leitstellenspiel675:157`
Sollte dieser behoben sein, steht einer implementierung in den LSSM von meiner seite aus nix im Weg.